src/styles/blocks/card.scss

```diff
@@ -0,0 +1,106 @@
+.card {
+  width: $card-width;
+  
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  padding: 32px 16px 16px;
+
+  &__image {
+    display: block;
+    height: 134px;
+    width: 160px;
+
+    margin: 0 auto 40px;
+  }
+
+  &__block-description {
+    margin-bottom: 4px;
+
+    @include font(500, 12px, 18px);
+    color: $color-main-accent;
+  }
+
+  &__block-product {
+    font-family: inherit;
+    margin-bottom: 16px;
+
+    @include font(400, 10px, 14px);
+    text-align: left;
+    color: $color-secondary-accent;
+  }
+
+  &__block-rating {
+    margin-bottom: 24px;
+
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  &__block-reviews {
+    font-family: inherit;
+
+    @include font(400, 10px, 14px);
+    color: $color-main-accent;
+  }
+
+  &__block-cost {
+    font-family: inherit;
+    margin-bottom: 16px;
+
+    display: flex;
+    justify-content: space-between;
+  }
+
+  &__block-price {
+    @include font(400, 12px, 18px);
+    color: $color-secondary-accent;
+  }
+
+  &__block-value {
+    @include font(700, 16px, 18px);
+    color: $color-main-accent;
+  }
+
+  &__block-link {
+    font-family: inherit;
+
+    display: block;
+    height: 40px;
+
+    padding: 12px 70px;
+
+    border-radius: 5px;
+    background-color: $color-accent-blue;
+    border: 1px solid transparent;
+    text-align: center;
+
+    @include font(700, 14px, 16px);
+    text-transform: uppercase;
+    color: #fff;
+
+    transition: 0.3s background-color ease,
+      0.3s color ease,
+      0.3s border-color ease;
+
+    &:hover {
+      background-color: transparent;
+      color: $color-accent-blue;
+      border-color: $color-accent-blue;
+    }
+  }
+}
+
+.catalog__card {
```

Comment:
This selector should be in the catalog file

Reply:
**FIxed**